services/nyx/custom_mutators/web_service_mutator/Makefile

```diff
@@ -0,0 +1,21 @@
+CC = gcc
```

Comment:
Is there a reason to use `gcc` specifically? Everything else is using `clang` (which would be in the environment if you didn't specify).